code/modules/client/preferences.dm

```diff
@@ -178,27 +178,42 @@ var/const/MAX_SAVE_SLOTS = 8
 
 	var/progress_bars = 1 //Whether to show progress bars when doing delayed actions.
 	var/client/client
+	var/loadingsave = 0
 	var/saveloaded = 0
 
 /datum/preferences/New(client/C)
 	client=C
 	if(istype(C))
 		var/theckey = C.ckey
+		var/thekey = C.key
```

Comment:
On the live server? It wouldn't do anything because guests are disallowed. But its needs to check key merely because ckey is always sanitized to be lowercase and whatever and it checks Guest.